src/components/Input/Input.ts

```diff
@@ -0,0 +1,51 @@
+import Block from '../../utils/Block';
+import template from './template.pug';
+import { VALIDATOR } from '../../utils/const'
+
+interface InputProps {
+  inputName: string;
+  inputValue?: string;
+  labelName?: string;
+  id: string;
+  errorText?: string;
+  type: string;
+  placeholder?: string;
+  required?:  boolean;
+}
+class Input extends Block {
```

Comment:
Так же по конвенции лучше делать отступы между объявлениями. Это лучше читается
```suggestion
}

class Input extends Block {
```